projects/angular-cld/src/lib/cloudinary-image.component.ts

```diff
@@ -121,22 +122,29 @@ export class CloudinaryImage
       image.onerror = e => {
         this.onError.emit(e);
       };
-      const options = this.cloudinary.toCloudinaryAttributes(
+      let options = this.cloudinary.toCloudinaryAttributes(
         nativeElement.attributes,
         this.transformations
       );
+
       if (this.clientHints || (typeof this.clientHints === 'undefined' && this.cloudinary.config().client_hints)) {
         delete options['class'];
         delete options['data-src'];
         delete options['responsive'];
       }
+      if (this.cloudinary.config().analytics) {
+        options = {...analyticsOptionsDefault, ...options};
```

Comment:
It's following the flat structure we have today. In core we only know what options are and I don't want to add there another object to look for.

Reply:
Why not? it's all under the same area

Reply:
Okay, we can have under a new object called analyticsOptions.

Reply:
@asisayag2  I think the point is that we never grouped our options and configurations by "area", this will be a first.

Certainly possible, but it's the first time done.

Reply:
Get what you mean